webapp/src/Entity/Contest.php

```diff
@@ -294,6 +294,14 @@ class Contest extends BaseApiEntity implements AssetEntityInterface
      */
     private bool $processBalloons = true;
 
+    /**
+     * @ORM\Column(type="boolean", name="order_by_runtime",
+     *     options={"comment"="Is runtime used as tiebreaker instead of penalty?","default"=0},
```

Comment:
I wonder if this should improve, since it's not only ordering by it but also showing it on the scoreboard. Not sure what's better though.

Reply:
I'd name it something like `runtime_as_score_tiebreaker`